converters/fromZigbee.js

```diff
@@ -1437,6 +1437,48 @@ const converters = {
             return {...payload1, ...payload2};
         },
     },
+    develco_voc: {
+        cluster: 'develcoSpecificAirQuality',
+        type: ['attributeReport', 'readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            const voc = parseFloat(msg.data['measuredValue']);
+            const vocProperty = postfixWithEndpointName('voc', msg, model);
+
+            let airQuality;
+            const airQualityProperty = postfixWithEndpointName('air_quality', msg, model);
+            if (voc <= 65) {
+                airQuality = 'excellent';
+            } else if (voc <= 220) {
+                airQuality = 'good';
+            } else if (voc <= 660) {
+                airQuality = 'moderate';
+            } else if (voc <= 2200) {
+                airQuality = 'poor';
+            } else if (voc <= 5500) {
+                airQuality = 'unhealthy';
+            } else if (voc > 5500) {
+                airQuality = 'out_of_range';
+            } else {
+                airQuality = 'unknown';
+            }
+            return {[vocProperty]: calibrateAndPrecisionRoundOptions(voc, options, 'voc'), [airQualityProperty]: airQuality};
+        },
+    },
+    develco_voc_battery: {
+        cluster: 'genPowerCfg',
+        type: ['attributeReport', 'readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            /*
+             * Per the technical documentation for AQSZB-110:
+             * To detect low battery the system can monitor the "BatteryVoltage" by setting up a reporting interval of every 12 hour.
+             * When a voltage of 2.5V is measured the battery should be replaced.
+             * Low batt LED indication–RED LED will blink twice every 60 second.
+             */
+            const result = converters.thermostat.convert(model, msg, publish, options, meta);
```

Comment:
I don't understand why the thermostat converter is called here. (should be the battery one?)

Reply:
Oops, indeed that should be battery.